### PR TITLE
Update kotlin to 1.5.20.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: jvb/src/main/java/org/jitsi/videobridge/cc/allocation/SingleSourceAllocation.java [hunk 1]
```diff
@@ -129,7 +129,8 @@
                     || (lessThanOrEqualIdealResolution && atLeastPreferredFps))
                     || ratesList.isEmpty())
             {
-                double layerBitrate = layer.getBitrate(nowMs);
+
+                double layerBitrate = layer.getBitrateBps(nowMs);
```
Comment (nils-ohlmeier) [inline jvb/src/main/java/org/jitsi/videobridge/cc/allocation/SingleSourceAllocation.java:133]: Are these code changes suppose to go into this PR?

**Review by nils-ohlmeier — APPROVED** [review]
LGTM - although I would recommend to move the code changes into a separate PR.

File: jvb/src/main/java/org/jitsi/videobridge/cc/allocation/SingleSourceAllocation.java (hunk 1 above)
Comment (bgrozev) [reply jvb/src/main/java/org/jitsi/videobridge/cc/allocation/SingleSourceAllocation.java:133]: Yes, they are required because with the updated jitsi-media-utils`RtpLayerDest.getBitrate()` doesn't get exported and can't be used in java (I don't know why, but this was the workaround I found).

File: pom.xml [hunk 2]
```diff
@@ -24,13 +24,14 @@
...
-        <kotlin.version>1.3.72</kotlin.version>
-        <kotest.version>4.1.3</kotest.version>
+        <kotlin.version>1.5.20</kotlin.version>
+        <kotest.version>4.6.1</kotest.version>
         <jicoco.version>1.1-89-g9ca691c</jicoco.version>
         <jitsi.utils.version>1.0-93-ge6aba2e</jitsi.utils.version>
         <maven-shade-plugin.version>3.2.2</maven-shade-plugin.version>
         <spotbugs.version>4.1.4</spotbugs.version>
         <jersey.version>2.34</jersey.version>
+        <jackson.version>2.12.4</jackson.version>
```
Comment (JonathanLennox) [inline pom.xml:34]: Why is this added?  I don't see it used anywhere?

Comment (nils-ohlmeier) [reply pom.xml:34]: > Why is this added? I don't see it used anywhere?

It is used in other pom.xml files further up.

**Review by JonathanLennox — APPROVED** [review]
